clusters/hive/pools/pipelines/pipelines-install-config.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pipelines-install-config
+  namespace: pipelines-cluster-pool
+type: Opaque
+stringData:
+  install-config.yaml: |-
+    apiVersion: v1
+    baseDomain: psi.ospqa.com
```

Comment:
is this the same PSI cluster which pipelines team using today ?

Reply:
Yes